netstandard/ref/System.Globalization.cs

```diff
@@ -333,7 +333,6 @@ public abstract partial class EastAsianLunisolarCalendar : System.Globalization.
         public override System.DateTime ToDateTime(int year, int month, int day, int hour, int minute, int second, int millisecond, int era) { throw null; }
         public override int ToFourDigitYear(int year) { throw null; }
     }
-    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public static partial class GlobalizationExtensions
     {
```

Comment:
Presumably we want to keep hiding this one as the type was a .NET Core 1.x helper that is now replace with the actual instance method on string.

Reply:
Yeah in general we probably don't want folks using these but I don't think it matters much as the compiler will prefer the instance methods anyway. I'm not sure if there is any real benefit either way.

Reply:
👍